### DISCUSSION TITLE
Why relayerXcm calls are disabled?

### DISCUSSION THREAD

**Discussion by dzmitry-lahoda** [AcalaNetwork/Acala#2385] [2. Q&A]
Hello, 

I see that I cannot call relayerXcm calls are disabled 

https://github.com/AcalaNetwork/Acala/blob/2b8b0533ecbd4e8b13d2e347ad8b186c4e358c1f/runtime/karura/src/lib.rs#L225

Can we enable them to be able to send complex multistep messages?

Rockmine/Statemine has XCM enabled.

Also I can use `send` and craft any message. So filtering out transfers in XCM does not prevents me from building cross chain messaging I want to do.

**Comment by xlc** [comment]
Can you provide some use cases of why you need them?

**Reply by dzmitry-lahoda** [reply]
i just using all networks, and all have unified interface (only asset ids are different - but that is easy).

**Reply by xlc** [reply]
That's not a good enough reason for us to make the change.

**Reply by dzmitry-lahoda** [reply]
I understand that as of now you decided not to allow more or less uniform interface for XCM. I am ok with that. Thank you
